modules/imgui/src/guijoystickcomponent.cpp

```diff
@@ -84,19 +86,21 @@ void GuiJoystickComponent::render() {
 
     ImGui::Text("%s", "Summed contributions");
     ImGui::Text("%s", "Axes");
-    for (int i = 0; i < JoystickInputState::MaxAxes; ++i) {
+    for (int i = 0; i < global::joystickInputStates->numAxes(""); ++i) {
```

Comment:
Should `numAxes` get the parameter with a default value instead so that we can write:
`global::joystickInputStates->numAxes()` here?